core/src/main/cpp/gdn/projection_mesh/equirectangular_projection_mesh.h

```diff
@@ -0,0 +1,34 @@
+#ifndef EQUIRECTANGULAR_PROJECTION_MESH_H
+#define EQUIRECTANGULAR_PROJECTION_MESH_H
+
+#include "projection_mesh.h"
+#include "projection_mesh_utils.h"
+
+namespace gast {
+
+namespace {
+using namespace godot;
+}
+
+class EquirectangularProjectionMesh : public ProjectionMesh {
+GODOT_CLASS(EquirectangularProjectionMesh, ProjectionMesh)
+
+public:
+    EquirectangularProjectionMesh();
+    ~EquirectangularProjectionMesh();
+
+    void _init();
+
+    void _process(const real_t delta);
+
+    static void _register_methods();
+
+    void update_projection_mesh();
```

Comment:
Is this meant to be public?

Reply:
update_projection_mesh should have been private, moved it.